src/partials/modal.html

```diff
@@ -0,0 +1,48 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
+    <div class="modal-top">
+      <button
+        type="button"
+        class="btn-burger header-icon-cross"
+        aria-expanded="false"
+        data-modal-close
+      >
+        <svg
+          class="header-icon-cross"
+          width="20"
+          height="20"
+          aria-label="Переклчатель мобильного меню"
+        >
+          <use class="header-icon-cross" xlink:href="../images/sprite.svg#cross"></use>
```

Comment:
Нужно заменить 2 точки на 1, как сделано на примере снизу:

```suggestion
          <use class="header-icon-cross" xlink:href="./images/sprite.svg#cross"></use>
```

Reply:
в том то и дело, что я не могла проверить это локально, выдавало ошибку. ментор сказал запушить, чтобы он посмотрел на ошибки. В итоге сказал эту ветку не трогать и создать новую, а эту потом удалим. Я создала, сделала всё то же самое, конфликт в итоге выдавали те две точки!! в этот раз я их не ставила и все пошло как надо. Запушила вторую ветку, 
пысы: не могу написать тебе в слак, нам написано, что сейчас мои сообщения не дойдут, так как ты в режиме  - не беспокоить